libsofia-sip-ua/soa/test_soa.c

```diff
@@ -2240,11 +2244,11 @@ int test_address_selection(struct context *ctx)
   TEST_OC_ADDRESS(a, "2001:1508:1003::21a:a0ff:fe71:813", ip6);
   TEST_VOID(soa_terminate(a, NULL));
 
-  /* SOATAG_AF(SOA_AF_IP4_IP6), o= mentions IP6 => select IP4  */
+  /* SOATAG_AF(SOA_AF_IP4_IP6) and SOATAG_USER_O_LINE() tag remove from so, o= mentions IP6 => select IP6  */
```

Comment:
We do not have `SOATAG_USER_O_LINE()` tag in our codebase. But we have it's functionality enabled by default.